### PR TITLE
fix: update json schema

### LINKED ISSUE #2894: JSON schema needs some updates
Current schema does not know about:
```toml
[[env]]    # <-- this
  
[tasks.a]
shell = 'zsh -c'   # <-- and this
```

### PR DESCRIPTION
**Pull request by hverlin**
Fix #2894

### CODE REVIEW

File: schema/mise.json
Comment (roele) [inline schema/mise.json:1]: AFAIR the `schema/mise.json` is generated from `schema/mise.json.hbs` by the `render:settings` task, so the changes should rather be made in `schema/mise.json.hbs`.

Comment (hverlin) [reply schema/mise.json:1]: Oh I see. I think one job should run the render job to make sure it got run before making the PR

**Review by roele — APPROVED** [review]

**Comment by powerman** [comment]
Just noticed `tasks.*.file` is missing from schema too. Should I open a new issue?

**Comment by hverlin** [comment]
Created https://github.com/jdx/mise/pull/2911

**Pull request merged**